tests/ossfuzz/config/fuzz_decode_alone.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+dict = fuzz_lzma.dict
```

Comment:
I believe the oss-fuzz will fall back to the default max length of the generated input of 4096 bytes if the `max_len` option is not provided in the `.options` file, so setting it explicitly is not necessary. My bad for not standardising the options file formats. 

We can either remove the files entirely as per the above comment, or include this option explicitly in each config file. Which options sounds better to you?

Reply:
Based on the [libFuzzer docs](https://llvm.org/docs/LibFuzzer.html#options), it seems that the default is a special 0 value that indicates the maximum input length should be inferred from the corpus. Most of our test files are small (I believe the largest is ~50 Kb .xz file) so its likely that the default value is good enough in practice.

However it could be a good idea to set an upper bound on a size we think is reasonable, especially for the fuzz targets that are meant to test handling bytes early in the input stream. So I would vote for this.

Reply:
Good! What size would you consider reasonable for the test cases?

Reply:
> Good! What size would you consider reasonable for the test cases?

The 4096 number you were using before seems good. That should be more than enough to trigger the sanitizers if there is a possible bad code path